ambry-clustermap/src/test/java/com.github.ambry.clustermap/AmbryStateModelFactoryTest.java

```diff
@@ -13,39 +13,54 @@
  */
 package com.github.ambry.clustermap;
 
+import com.codahale.metrics.MetricRegistry;
 import com.github.ambry.config.ClusterMapConfig;
 import com.github.ambry.config.VerifiableProperties;
+import com.github.ambry.utils.TestUtils;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Properties;
+import org.apache.helix.model.Message;
 import org.apache.helix.participant.statemachine.StateModel;
+import org.json.JSONObject;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
+import org.mockito.Mockito;
 
+import static com.github.ambry.clustermap.TestUtils.*;
 import static org.junit.Assert.*;
+import static org.junit.Assume.*;
+import static org.mockito.Mockito.*;
 
 
 /**
- * Test for {@link AmbryStateModelFactory}
+ * Test for {@link AmbryStateModelFactory} and {@link AmbryPartitionStateModel}
  */
 @RunWith(Parameterized.class)
 public class AmbryStateModelFactoryTest {
   private final ClusterMapConfig config;
+  private final String stateModelDef;
 
   @Parameterized.Parameters
   public static List<Object[]> data() {
     return Arrays.asList(
         new Object[][]{{ClusterMapConfig.OLD_STATE_MODEL_DEF}, {ClusterMapConfig.AMBRY_STATE_MODEL_DEF}});
   }
 
-  public AmbryStateModelFactoryTest(String stateModelDef) {
+  public AmbryStateModelFactoryTest(String stateModelDef) throws Exception {
+    List<com.github.ambry.utils.TestUtils.ZkInfo> zkInfoList = new ArrayList<>();
```

Comment:
you can just say ZkInfo since you are already importing TestUtils.*

Reply:
fixed